lib/nailed/bugzilla.rb

```diff
@@ -8,10 +8,13 @@ class Bugzilla
     def initialize
       Nailed::Config.parse_config
       Bicho.client = Bicho::Client.new(Nailed::Config.content["bugzilla"])
+      @products = Nailed::Config.products.collect{
+        |x| Nailed::Config.combined.fetch(x, x)
+      }.flatten
```

Comment:
multiline blocks should be with `do ... end` and the `|x|` should be at the end of the first line